leits/MeetingBar
# feat(services): add support for livestorm webinars

### Status
**READY**

### Description

This MR adds support for livestorm webinars

### IMPORTANT
This branch is based on the branch with chorus & gong support. Please merge this MR #425 first.

## Checklist

### Steps to Test

* Create a calendar entry with following url https://app.livestorm.co/p/cc113fd5-5de1-406-ba74-85c4892530/live?s=0231a8fb-fce9-48b0-9263-525f4234234234
* Check that meeting is recognized for livestorm with the specified icon
